wire-ios-data-model/Source/Model/Message/ConversationMessage.swift

```diff
@@ -31,12 +31,6 @@ public enum ZMDeliveryState: UInt {
     case failedToSend = 5
 }
 
-@objc
```

Comment:
This has been replaced by `ExpirationReason` as it is not necessarily a `failure`